packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2700,7 +2700,10 @@ export class ContainerRuntime
 				this.updateDocumentDirtyState(false);
 			}
 
-			this.validateAndProcessRuntimeMessage(messageWithContext, localOpMetadata);
+			// validateAndProcessRuntimeMessage only processes internal container messages
+			if (messageWithContext.message.type in ContainerMessageType) {
```

Comment:
Please take a look at the `default` case in `validateAndProcessRuntimeMessage`, there's all the stuff about compatDetails.

I do plan to remove it, but haven't yet, so please keep the behavior as-is (hopefully tests fail with the PR in its current state...?)